ckan/lib/helpers.py

```diff
@@ -2344,6 +2344,35 @@ def mail_to(email_address, name):
     return html
 
 
+@core_helper
+def get_base_public_folder(app_conf):
```

Comment:
The functions are used in multiple files, so I thought `helpers.py` is the right location to include them into.

What do you suggest as other location?